prediction-applier/src/main/java/org/alfresco/hxi_connector/prediction_applier/model/prediction/PredictionBatch.java

```diff
@@ -25,20 +25,10 @@
  */
 package org.alfresco.hxi_connector.prediction_applier.model.prediction;
 
-import java.util.Map;
-
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
 public record PredictionBatch(
-        @JsonProperty("_id") String id,
-        String modelId,
-        String fieldConfigurationId,
-        String field,
-        String enrichmentType,
-        Double threshold,
-        @JsonProperty("dateCreated") String creationDate,
-        String status,
-        Integer currentPage,
```

Comment:
I would have expected us to need the current page (e.g. if restarting the application). Does it work to just ask HxI for the latest page?
Don't we need the other properties? (e.g. `field` seems essential and we store `enrichmentType` in the ACS content model).

Reply:
I think all data we need is also stored (field, enrichmentType) in individual prediction entry, so we don't use these fields. Also "currentPage" property doesn't tell us anything as we don't have number of all pages🤦‍♂️ - so we cannot resume processing if app will fail because the only info we have about every batch is: status "COMPLETE", page: 5, but we don't know if there is just 5 pages or there's more of them

Reply:
I thought that we would be able to see:
* IN_PROGRESS, page 5: We need to process page 5
* COMPLETE, page 5: We have processed all pages (and the last page was page 5, or maybe 4)

If the application goes down part way through and we see IN_PROGRESS, page 5 then we may accidentally process the same page twice, but at least we shouldn't lose data like this.

Reply:
yep, you're right, I restored currentPage property, I think status is not needed as we always want to query for batches in some status so we already know it